portfolio/src/main/webapp/script.js

```diff
@@ -17,9 +17,62 @@ function init_index() {
 }
 
 function init_games() {
+    getLogin();
     getGameTabs();
 }
 
+function getLogin() {
+    var loginButton = document.getElementById('game-login-button');
+    var loginText = document.getElementById('game-login-text');
+
+    fetch('/login').then(response => response.text()).then((response) => {
+      var responseSplit = response.split("\n");
+      var message = responseSplit[0];
+      var url = responseSplit[1];
+      console.log(url);
+      console.log(message + "  " + url);
+      loginButton.href = url;
+      loginText.innerText = message;
+
+      if (message.charAt(0) == "H") {
```

Comment:
Same as comment above, highly suggest not relying on line or character positioning for decision making.  eg. It could be a nightmare to debug if someone changed the message from "Hello" to "What's Up", etc..